cmd/azure.go

```diff
@@ -49,30 +62,38 @@ func azureToObjectError(err error, params ...string) error {
 	if len(params) == 2 {
 		object = params[1]
 	}
-	actualErr := err
-	traceErr, isTraceErr := err.(*Error)
-	if isTraceErr {
-		actualErr = traceErr.e
-	}
 
-	azureErr, ok := actualErr.(storage.AzureStorageServiceError)
-	if !ok {
+	azureErr, az := err.(storage.AzureStorageServiceError)
+	if !az {
```

Comment:
better to have az as ok, and rename ok as IsTraceErr (like how it was previously) as a generic ok here spans here for 3 lines of code, and more descriptive isTraceErr spans more lines of code and hence makes code more readable.